CHANGELOG.md

```diff
@@ -8,19 +8,22 @@ The public API of this library consists of the functions declared in file
 ## [Unreleased]
 ### Added
 - `experimentalH3ToLocalIj` function for getting local coordinates for an index. (#102)
-### Fixed
-- Added #include <stdio.h> to benchmark.h
-### Added
+- `experimentalLocalIjToH3` function for getting an index from local coordinates. (#102)
 - Benchmarks for the kRing method for k's of size 10, 20, 30, and 40.
+### Changed
+- Internal `h3ToIjk` function renamed to `h3ToLocalIjk`. (#102)
```

Comment:
Is this supposed to be `h3ToLocalIj` like elsewhere or no?

Reply:
No, as below the IJK functions need to be kept for internal implementation.